src/SFA.DAS.ApplyService.InternalApi.UnitTests/ExperienceAndAccreditationControllerTests.cs

```diff
@@ -0,0 +1,177 @@
+﻿using NUnit.Framework;
+using AutoMapper;
+using SFA.DAS.ApplyService.InternalApi.AutoMapper;
+using SFA.DAS.ApplyService.Application.Apply;
+using Moq;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.ApplyService.InternalApi.Controllers;
+using SFA.DAS.ApplyService.InternalApi.Infrastructure;
+using SFA.DAS.ApplyService.InternalApi.Models.Ukrlp;
+using SFA.DAS.ApplyService.Domain.Ukrlp;
+using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using SFA.DAS.ApplyService.Application.Apply.Roatp;
+using SFA.DAS.ApplyService.Domain.Apply;
+using SFA.DAS.ApplyService.Domain.Entities;
+using SFA.DAS.ApplyService.Domain.Roatp;
+using SFA.DAS.ApplyService.InternalApi.Types.CompaniesHouse;
+using SFA.DAS.ApplyService.InternalApi.Types;
+using SFA.DAS.ApplyService.InternalApi.Services;
+
+namespace SFA.DAS.ApplyService.InternalApi.UnitTests
+{
+    [TestFixture]
+   public class ExperienceAndAccreditationControllerTests
```

Comment:
There seems to be quite a lot of repetition in the class.  Could some of it be moved the SetUp or abstracted out into methods that are reused?